src/com/owncloud/android/ui/activity/FileDisplayActivity.java

```diff
@@ -209,7 +212,6 @@ protected void onCreate(Bundle savedInstanceState) {
     protected void onStart() {
         Log_OC.v(TAG, "onStart() start");
         super.onStart();
-        getSupportActionBar().setIcon(DisplayUtils.getSeasonalIconId());
```

Comment:
Yes, the seasonal icon a a method to determine which app icon to load (standard or winter/snowy one).